packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -32,7 +32,102 @@ class Plugin {
     this.pluginClass,
   });
 
+  /// Parses [Plugin] specification from the provided pluginYaml.
+  ///
+  /// This currently supports two formats. V1 (Legacy) and V2 (Multi-platform).
+  /// Example V1 format. Which is deprecated.
```

Comment:
`V1` and `V2` might be confusing. Might be better to stick with just `Legacy` and `MultiPlatform` as the canonical names for these.

Reply:
legacy and multiplat sounds reasonable.